tests/benchdnn/inputs/matmul/test_matmul_random_ci

```diff
@@ -0,0 +1,400 @@
+--reset --allow-enum-tags-only=0 --stag=bac --wtag=cba --dtag=cab --dt=f8_e4m3:f8_e4m3:bf16  1x362x16:1x16x12
```

Comment:
`--allow-enum-tags-only=0` is fine for converter but not for a generator. It shouldn't run beyond the library capabilities.